src/zc/lockfile/__init__.py

```diff
@@ -102,3 +103,37 @@ def close(self):
             _unlock_file(self._fp)
             self._fp.close()
             self._fp = None
+            
+class LockedFile(object):
+    __slots__ = ("name", "mode", "args", "kwargs", "lock", "file", "timeout")
```

Comment:
Why are slots necessary here?

Reply:
They aren't. I was copied this patch from another project I am working on, where slots are part of the local style guide. I can remove them if you'd like.

Reply:
They aren't. I removed them.